tutorials/interactive/index.md

```diff
@@ -0,0 +1,781 @@
+---
+layout: tutorial
+title: "Adding Interactivity to your Flutter App"
+sidebar: home_sidebar
+permalink: /tutorials/interactive/
+---
+
+{% comment %}
+TODO:
+
+* Restore _private variables in tapbox examples.
+* Add the following examples to the GitHub repo, and update all links
+  * lakes-interactive
+  * tapbox-a
+  * tapbox-b
+  * tapbox-c
+{% endcomment %}
+
```

Comment:
Perhaps for a "what you'll learn"

* Respond to taps
* Create a custom widget
* The difference between a stateless and stateful widget